packages/vite/src/modes/global/dev.ts

```diff
@@ -164,7 +164,7 @@ export function GlobalModeDevPlugin({ uno, tokens, affectedModules, onInvalidate
         // inject css modules to send callback on css load
         if (layer && code.includes('import.meta.hot')) {
           return `${code}
-if (import.meta.hot) {
+if (import.meta.hot && import.meta.hot.send) {
```

Comment:
If `viteHotContext` is created successfully, then `import.meta.hot.send` must exist, which is an invalid judgment. I don't think the problem is here🤔️

Reply:
I've looked into it and debug , sometimes `import.meta.hot.send` were `undefined`

Reply:
Unbelievable 😅

Reply:
So was i 🤣

Reply:
This API is introduced in Vite 2.9, which UnoCSS requires it. If this is `undefined` I think it's probably because they have older vite installed.

Reply:
> 
[#1823](https://github.com/winchesHe/Idux-interactive-histoire/blob/main/package.json#L30) provide the reproduction vite version was `^3.1.6`